Update TAG Security Documents: README and Charter (#1316)
cncf/tag-security: README.md

<!-- cSpell:disable -->

## Leadership

> TAG Chairs, Tech Leads and TOC Liaisons are listed on the [CNCF Technical Advisory Groups ("TAGs") information page](https://github.com/cncf/toc/blob/main/tags/cncf-tags.md).

Reviewer: ```suggestion
Details about the TAG Chairs, Tech Leads, and TOC Liaisons can be found on the [CNCF Technical Advisory Groups (TAGs) information page](https://github.com/cncf/toc/blob/main/tags/cncf-tags.md)
```

create a
new [security review issue](https://github.com/cncf/tag-security/issues/new?assignees=&labels=assessment&template=security-assessment.md&title=%5BAssessment%5D+Project+Name)
with a
[self-assessment](/community/assessments/guide/self-assessment.md).

## TAG Emeritus Leaders

> A big thank you to all the [tag emeritus leaders](/tag-emeritus-leaders.md) of this TAG! Your hard work and dedication have helped to make this project a success. Your valuable contributions have enabled us to develop a strong contributor strategy and build a thriving open-source community. Thank you for all that you have done!

Reviewer: The statement in this section feels overly sentimental and doesn't add any value. It is also not consistent with the tone of the rest of the README. Consider removing or revising it to be more concise and focused, especially since leadership is already linked elsewhere.

Author: Good point, agreed.

with a
[self-assessment](/community/assessments/guide/self-assessment.md)
.

> 
> ## TAG Emeritus Leaders
> <!-- cSpell:disable -->

Reviewer 2: ```suggestion
<!-- cSpell:disable -->

## TAG Emeritus Leaders
```

Reviewer 2: I don't know why this would work, but changing it to match the previous implementation
